novawallet/Common/Network/Subquery/Models/SubqueryHistory.swift

```diff
@@ -52,6 +59,7 @@ struct SubqueryHistoryElement: Decodable {
         case extrinsic
         case transfer
         case assetTransfer
+        case poolReward
```

Comment:
Note that we should ensure this key wont be included in request for networks without a pool, otherwise request will fail